doc/api_base.rst

```diff
@@ -191,7 +191,7 @@
   .. method:: install(pkg_spec, reponame=None, strict=True, forms=None)
 
     Mark packages matching `pkg_spec` for installation. 
-    Parameter `reponame` could be a name of repository or a list or repository names. If given, the selection of available packages is limited to packages from these repositories. If strict is set to False, the installation ignores packages with dependency solving problems. Parameter `forms` has the same meaning as in :meth:`dnf.subject.Subject.get_best_query`.
+    `Reponame` can be a name of a repository or a list of repository names. If given, the selection of available packages is limited to packages from these repositories. If strict is set to False, the installation ignores packages with dependency solving problems. Parameter `forms` has the same meaning as in :meth:`dnf.subject.Subject.get_best_query`.
```

Comment:
I think `reponame` should start with a lowercase, since it's an identifier.

Reply:
Fixed